hazelcast-sql-core/src/main/java/com/hazelcast/sql/impl/calcite/validate/HazelcastSqlOperatorTable.java

```diff
@@ -428,4 +181,59 @@ private HazelcastSqlOperatorTable() {
     public static HazelcastSqlOperatorTable instance() {
         return INSTANCE;
     }
+
+    /**
+     * Visitor that rewrites Calcite operators with operators from this table.
+     */
+    public static class RewriteVisitor extends SqlBasicVisitor<Void> {
+
+        private final HazelcastSqlValidator validator;
```

Comment:
Field is unused

Reply:
Fixed in https://github.com/hazelcast/hazelcast/pull/17947/commits/65df331cf8df7c9690ee4f961f2a1d81a905fdfc